creator-node/src/utils/registeredNodes.ts

```diff
@@ -0,0 +1,65 @@
+import type Logger from 'bunyan'
```

Comment:
can we make filename more specific e.g. `getRegisteredDiscoveryNodes`

Reply:
i was thinking of moving the getRegisteredContentNodes in this file too... but yeah let me rename the file

Reply:
yeah for now lets just rename and leave separate thx